.ci/gitlab/test.yml

```diff
@@ -204,6 +204,15 @@ suite:vivado:verilog:
     - local
     - vivado-2022.1-standard
 
+suite:vivado:systemverilog:
```

Comment:
Should `suite:systemverilog` be renamed to `suite:verilator:systemverilog`?

Reply:
Could you add a comment mentioning that we run Vivado/SV as a separate job due to performance (I'm assuming that's why)?

Reply:
We already have that:
https://github.com/clash-lang/clash-compiler/blob/f166a204da010e267b8653680c193209c5c1fec6/.ci/gitlab/test.yml#L188-L189
Is this insufficient?

Reply:
> Should suite:systemverilog be renamed to suite:verilator:systemverilog?

I think it's more of a coincidence that we only run Verilator on SystemVerilog; it's not a design decision, it's simply the case because we never made the effort to install Questa on our runners.

Conversely, running Vivado in a separate job is truly a design decision.

As far as I'm concerned, some day soon we install Questa Intel FPGA Starter Edition on all our runners and run that as part of `suite:systemverilog`. I think it's fast enough to run both Verilator and Questa as part of every CI run. It's just Vivado that's such a 🐢.

Reply:
Ah sorry, nothing to be done here then.